sprint7/tema3.js

```diff
@@ -0,0 +1,35 @@
+function getDay(n){
+    let dayName;
+
+    switch (n){
+        case 0:
```

Comment:
```suggestion
        case 0, 7:
```

e binevenit 7 pentru duminica